tests/oss-fuzz/build.sh

```diff
@@ -123,10 +125,7 @@ chmod -x \$this_dir/$fuzz_basename" > $OUT/$TARGET_FUZZER
   done
 fi
 
-# copy seed corpus for fuzztest tests
-mkdir $OUT/corpus
-unzip $SRC/avif_decode_seed_corpus.zip -d $OUT/corpus
-cp $SRC/libavif/tests/data/* $OUT/corpus
-
 # create a bigger seed corpus for avif_decode_fuzzer
-zip -j $OUT/avif_decode_fuzzer_seed_corpus.zip $OUT/corpus/*
+cp $SRC/avif_decode_seed_corpus.zip $OUT/avif_decode_fuzzer_seed_corpus.zip
+zip -j $OUT/avif_decode_fuzzer_seed_corpus.zip \
+  $(find $SRC/libavif/tests/data -maxdepth 1 -type f)
```

Comment:
I verified the contents of the zip file contain only the files from the original and the top-level of tests/data with no top-level directory. This does include README.md, but that's fairly harmless.

Reply:
> I verified the contents of the zip file contain only the files from the original and the top-level of tests/data with no top-level directory. This does include README.md, but that's fairly harmless.

Perhaps the format did change or I missed something else:

```
=================================================================
==496==ERROR: AddressSanitizer: ABRT on unknown address 0x0000000001f0 (pc 0x7f8ba7ea100b bp 0x7fffa44f8530 sp 0x7fffa44f8190 T0)
SCARINESS: 10 (signal)
    #0 0x7f8ba7ea100b in raise (/lib/x86_64-linux-gnu/libc.so.6+0x4300b) (BuildId: eebe5d5f4b608b8a53ec446b63981bba373ca0ca)
    #1 0x7f8ba7e80858 in abort (/lib/x86_64-linux-gnu/libc.so.6+0x22858) (BuildId: eebe5d5f4b608b8a53ec446b63981bba373ca0ca)
    #2 0x5db7be in avif::testutil::GetTestImagesContents(unsigned long, std::__1::vector<avifAppFileFormat, std::__1::allocator<avifAppFileFormat> > const&) /src/libavif/tests/gtest/avif_fuzztest_helpers.cc:276:5
    #3 0x51a552 in avif::testutil::ArbitraryImageWithSeeds(std::__1::vector<avifAppFileFormat, std::__1::allocator<avifAppFileFormat> > const&) /src/libavif/tests/gtest/avif_fuzztest_helpers.h:230:18
    #4 0x444301 in __cxx_global_var_init.1 /src/libavif/tests/gtest/avif_fuzztest_dec.cc:39:18
    #5 0x444301 in _GLOBAL__sub_I_avif_fuzztest_dec.cc /src/libavif/tests/gtest/avif_fuzztest_dec.cc
    #6 0x32c1a9c in __libc_csu_init (/tmp/not-out/tmpzu_z0_8u/avif_fuzztest_dec+0x32c1a9c)
    #7 0x7f8ba7e8200f in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x2400f) (BuildId: eebe5d5f4b608b8a53ec446b63981bba373ca0ca)
    #8 0x4494ed in _start (/tmp/not-out/tmpzu_z0_8u/avif_fuzztest_dec+0x4494ed)

DEDUP_TOKEN: raise--abort--avif::testutil::GetTestImagesContents(unsigned long, std::__1::vector<avifAppFileFormat, std::__1::allocator<avifAppFileFormat> > const&)
AddressSanitizer can not provide additional info.
SUMMARY: AddressSanitizer: ABRT (/lib/x86_64-linux-gnu/libc.so.6+0x4300b) (BuildId: eebe5d5f4b608b8a53ec446b63981bba373ca0ca) in raise
```

Reply:
> > I verified the contents of the zip file contain only the files from the original and the top-level of tests/data with no top-level directory. This does include README.md, but that's fairly harmless.
> 
> Perhaps the format did change or I missed something else:
> 
> ```
> =================================================================
> ==496==ERROR: AddressSanitizer: ABRT on unknown address 0x0000000001f0 (pc 0x7f8ba7ea100b bp 0x7fffa44f8530 sp 0x7fffa44f8190 T0)
> SCARINESS: 10 (signal)
>     #0 0x7f8ba7ea100b in raise (/lib/x86_64-linux-gnu/libc.so.6+0x4300b) (BuildId: eebe5d5f4b608b8a53ec446b63981bba373ca0ca)
>     #1 0x7f8ba7e80858 in abort (/lib/x86_64-linux-gnu/libc.so.6+0x22858) (BuildId: eebe5d5f4b608b8a53ec446b63981bba373ca0ca)
>     #2 0x5db7be in avif::testutil::GetTestImagesContents(unsigned long, std::__1::vector<avifAppFileFormat, std::__1::allocator<avifAppFileFormat> > const&) /src/libavif/tests/gtest/avif_fuzztest_helpers.cc:276:5
>     #3 0x51a552 in avif::testutil::ArbitraryImageWithSeeds(std::__1::vector<avifAppFileFormat, std::__1::allocator<avifAppFileFormat> > const&) /src/libavif/tests/gtest/avif_fuzztest_helpers.h:230:18
>     #4 0x444301 in __cxx_global_var_init.1 /src/libavif/tests/gtest/avif_fuzztest_dec.cc:39:18
>     #5 0x444301 in _GLOBAL__sub_I_avif_fuzztest_dec.cc /src/libavif/tests/gtest/avif_fuzztest_dec.cc
>     #6 0x32c1a9c in __libc_csu_init (/tmp/not-out/tmpzu_z0_8u/avif_fuzztest_dec+0x32c1a9c)
>     #7 0x7f8ba7e8200f in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x2400f) (BuildId: eebe5d5f4b608b8a53ec446b63981bba373ca0ca)
>     #8 0x4494ed in _start (/tmp/not-out/tmpzu_z0_8u/avif_fuzztest_dec+0x4494ed)
> 
> DEDUP_TOKEN: raise--abort--avif::testutil::GetTestImagesContents(unsigned long, std::__1::vector<avifAppFileFormat, std::__1::allocator<avifAppFileFormat> > const&)
> AddressSanitizer can not provide additional info.
> SUMMARY: AddressSanitizer: ABRT (/lib/x86_64-linux-gnu/libc.so.6+0x4300b) (BuildId: eebe5d5f4b608b8a53ec446b63981bba373ca0ca) in raise
> ```

The tests want the data in corpus/ too.